hack/pre-release.sh

```diff
@@ -303,6 +318,92 @@ dist_git_update() {
   echo
 }
 
+# This function handles updating the versions in gitlab. Expects you to be connected to the VPN
+# takes 2 arguments
+# 1st arg: stream to update. Can be x, y or z to represent the x y or z stream.
+# 2nd arg: current WMCO version
+# 3rd arg: updated WMCO version
+gitlab_update() {
+ if [ "$#" -ne 3 ]; then
+    echo incorrect parameter count for dist_git_update $#
+    return 1
+  fi
+  stream=$1
```

Comment:
is this arg used? I would expect it to be -- are the the gitlab steps for x stream vs y/z all the same?

Reply:
I use it for naming the branch, but I could remove it if we want

Reply:
I see, we should leave it them. What I was trying to imply here was that it'll be used to see if we have to `Change CDN repo number to correct one for release binary` (x stream only, which will be handled later in the story you're creating)